azure-pipelines.yml

```diff
@@ -75,6 +80,29 @@ jobs:
         **\WebJobs.Script.csproj
         **\WebJobs.Script.WebHost.csproj
         **\WebJobs.Script.Grpc.csproj
+  
+  - pwsh: |
+      foreach ($baseName in @("WebJobs.Script", "WebJobs.Script.WebHost", "WebJobs.Script.Grpc"))
+      {    
+        $packageName = "Microsoft.Azure." + $baseName + "*.nupkg"    
+        $sourcePath = "$(Build.Repository.LocalPath)/packages/$packageName"        
+        if (-not (test-path $sourcePath))    
+        {        
+          throw "Unable to find '$packageName' at './package'"    
+        }   
+        Copy-Item -Path $sourcePath -Destination $(Build.ArtifactStagingDirectory) -ErrorAction Stop -Verbose -Force}
+    condition: eq(variables['RUNBUILDFORINTEGRATIONTESTS'], 'True')
+    displayName: 'Copy package to ArtifactStagingDirectory'
+
+  - task: NuGetCommand@2
+    condition: eq(variables['RUNBUILDFORINTEGRATIONTESTS'], 'True')
+    inputs:
+      command: 'push'
+      packagesToPush: '$(Build.ArtifactStagingDirectory)/**/*.nupkg;!$(Build.ArtifactStagingDirectory)/**/*.symbols.nupkg'
+      nuGetFeedType: 'internal'
+      publishVstsFeed: 'e6a70c92-4128-439f-8012-382fe78d6396/f37f760c-aebd-443e-9714-ce725cd427df'
```

Comment:
What are this GUIDS?

Reply:
The first is the GUID for the AzureFunctionsPreRelease feed - I am trying to find what the second is for.